Formula/gitversion.rb

```diff
@@ -0,0 +1,26 @@
+class Gitversion < Formula
+  desc "Easy semantic versioning for projects using Git"
+  homepage "https://github.com/GitTools/GitVersion"
+  url "https://www.nuget.org/api/v2/package/GitVersion.CommandLine/3.6.2"
+  version "3.6.2"
+  sha256 "db05ebacfb76ddca26180f0c70060fbb89800e434d3d55b524785d0da37dbac8"
+
+  bottle :unneeded
+
+  depends_on "mono"
+
+  def install
+    file = File.new("gitversion", "w", 0555)
+    file.write("#!/bin/sh\n")
+    file.write("#{HOMEBREW_PREFIX}/opt/mono/bin/mono #{libexec}/tools/GitVersion.exe \"$@\"\n")
+    file.flush
+    file.close
+
+    libexec.install Dir["*"]
+    bin.install_symlink "#{libexec}/gitversion"
+  end
+
+  test do
+    system "#{bin}/gitversion", "-?"
```

Comment:
I would love for GitVersion to do something more substantial, but that would require a git repository. GitVersion itself isn't really dependent on Git, so I would prefer if only the bottle could depend on Git in library form to do the following:

1. Create a temporary folder somewhere.
2. Initialize a Git repository in the temporary folder.
3. Create a text file in the temporary folder.
4. Commit the text file without warnings or errors about invalid author information. It looks like tests are executed under a separate user and I'd like the test to succeed even if people haven't configured the global Git user.
5. Run `gitversion` in the temporary folder and assert that its `stdout` contains `"FullSemVer":"0.1.0+0"`.

Is this at all possible? An alternative could be to bundle a Git repository with the bottle in a `.gz` or something, but I don't know how to best approach that solution.